test/common/performance/image_helpers.go

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package performance
+
+import (
+	"flag"
+	"knative.dev/eventing/test/common/performance/aggregator"
+	"knative.dev/eventing/test/common/performance/receiver"
+	"knative.dev/eventing/test/common/performance/sender"
+	"knative.dev/pkg/signals"
```

Comment:
Imports again :) (sorry!)

Reply:
I'm wondering, there is any go fmt option that does the imports grouping automatically?

Reply:
`goimports` does it if I’m not mistaken